changes.go

```diff
@@ -161,11 +161,10 @@ func (c *pollChangeManager) getChanges(changesUri *url.URL) error {
 	log.Debugf("Fetching changes: %s", uri)
 
 	/* If error, break the loop, and retry after interval */
-	client := &http.Client{Timeout: httpTimeout} // must be greater than block value
 	req, err := http.NewRequest("GET", uri, nil)
 	addHeaders(req)
-
-	r, err := client.Do(req)
+	httpclient.CheckRedirect = nil
```

Comment:
Why are you doing this? I think you should be able to leave it in for all, can't you?